src/devtools/client/debugger/src/components/shared/Popover.js

```diff
@@ -57,7 +57,8 @@ class Popover extends Component {
   }
 
   onTimeout = () => {
-    const isHoveredOnGap = this.$gap && window.elementIsHovered(this.$gap);
+    const isHoveredOnGap =
+      this.$gap && [...this.$gap.firstElementChild.classList].includes("hovered");
```

Comment:
I'm not a big fan of using these `onMouseEnter` `onMouseLeave` things. Is it possible to use `.matches(':hover')` instead?

Reply:
oh boy i love it when this happens. yep that got it down to a single line.